screen/wallets/CreateRecoveryPhraseScreen.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text, StyleSheet, TouchableOpacity } from 'react-native';
+import { useExtendedNavigation } from '../../hooks/useExtendedNavigation';
+import { RouteProp, useRoute } from '@react-navigation/native';
+import { DetailViewStackParamList } from "../../navigation/DetailViewStackParamList";
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { CommonActions } from '@react-navigation/native';
+import { OnboardingStackParamList } from '../../navigation/OnboardingStack';
+
+type NavigationProps = NativeStackNavigationProp<DetailViewStackParamList, 'WalletsList'>;
+
+const CreateRecoveryPhraseScreen = () => {
+    const { dispatch } = useExtendedNavigation<NavigationProps>();
+    const route = useRoute<RouteProp<OnboardingStackParamList, 'CreateRecoveryPhrase'>>();
+    const { walletID } = route.params;
+
+
+    const handleContinue = () => {
+        // Navigate to AddWalletRoot's PleaseBackup screen to show seed phrase
+        dispatch(
+            CommonActions.reset({
+                index: 0,
+                routes: [
+                    {
+                        name: 'AddWalletRoot',
+                        state: {
+                            routes: [{ name: 'PleaseBackup', params: { walletID } }],
+                        },
+                    },
+                ],
+            }),
+        );
+    };
+    return (
+        <View style={styles.container}>
+            <Text style={styles.title}>First, let's create your recovery phrase</Text>
+            <Text style={styles.description}>
+                A recovery phrase is a series of 12 words in a specific order. This word combination is unique to your wallet. Make sure to have pen and paper ready so you can write it down.
+            </Text>
+            <TouchableOpacity style={styles.button} onPress={handleContinue}>
+                <Text style={styles.buttonText}>Continue</Text>
```

Comment:
nit
```suggestion
                <Text style={styles.buttonText}>Create my wallet</Text>
```

and on the onboarding screen, change the button text to 'Continue'